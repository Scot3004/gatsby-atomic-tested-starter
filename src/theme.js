const theme = {
  backgroundColor: '#f5f5f5',
  textColor: '#000',
  mutedColor: '#555',
  primaryColor: '#663399',
  visitedColor: '#402060'
};

export default theme;
