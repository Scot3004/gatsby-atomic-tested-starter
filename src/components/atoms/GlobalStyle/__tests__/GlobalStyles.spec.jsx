import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import theme from '../../../../theme';
import GlobalStyles from '..';

describe('When I visit the site', () => {
  const tree = renderer.create(<GlobalStyles theme={theme} />).toJSON();
  test('Then the global styles should be added to the site', () => {
    expect(tree).toMatchSnapshot();
  });
});
