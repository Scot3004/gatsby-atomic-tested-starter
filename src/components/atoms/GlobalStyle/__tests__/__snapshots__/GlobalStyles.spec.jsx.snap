// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When I visit the site Then the global styles should be added to the site 1`] = `null`;
